helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/InstanceAccessor.java

```diff
@@ -315,26 +316,27 @@ public Response getInstanceConfig(@PathParam("clusterId") String clusterId,
     return notFound();
   }
 
-  @PUT
+  @POST
```

Comment:
PUT is for "set" and POST is for "patch", I'd suggest we keep both. @dasahcc thoughts?